src/cdk-experimental/menu/menu.ts

```diff
@@ -72,7 +73,10 @@ export class CdkMenu extends CdkMenuGroup implements Menu, AfterContentInit, OnI
   @Output() readonly closed: EventEmitter<void | 'click' | 'tab' | 'escape'> = new EventEmitter();
 
   /** Track the Menus making up the open menu stack. */
-  _menuStack: MenuStack;
+  // We provide a default MenuStack implementation in case the menu is an inline menu.
+  // For Menus part of a MenuBar nested within a MenuPanel this will be overwritten
+  // to the correct parent MenuStack.
```

Comment:
Adding implementation comments between a JsDoc and a member will break the connection